libs/designsystem/src/lib/components/button/button.component.ts

```diff
@@ -30,8 +33,9 @@ export class ButtonComponent implements AfterContentInit {
   isAttentionLevel2: boolean;
   @HostBinding('class.attention-level3')
   isAttentionLevel3: boolean;
-  @HostBinding('class.attention-level4')
-  isAttentionLevel4: boolean;
+  @HostBinding('class.no-decoration')
+  hasNoDecoration = false;
```

Comment:
```suggestion
  @HostBinding('class.no-decoration')
  @Input() set hasNoDecoration(enable: boolean) {
    this.isAttentionLevel1 = !this.enable; 
    this.isAttentionLevel2 = false;
    this.isAttentionLevel3 = false;
  }
```

Reply:
Consider something like this instead. That way we might be able to combine the hostbinding and the `@Input() set noDecoration(enable: boolean)` further down.

Reply:
-simplified the input parameter to work like <kirby-button noDes
coration ...>  - specifying the true false makes very little or no sense.